### ISSUE TITLE
Issue with Rasa Engine

### ISSUE THREAD

**Issue by rtk29** [IBM/nl2ltl#24]
Hi,

I have some problems with using the Rasa Engine for my use case. The error is "ModelNotFound: Model None can not be loaded." 

Can you please help me with resolving this issue ?

The Note in the readme file says that a tar file is required for this. But I am not able to fix this issue.

**Comment by francescofuggitti** [comment]
Hi, @rtk29!

In the repo, under the `nl2ltl/engines/rasa/models/` folder, there is a pre-trained model `nlu-2023...curve.tar.gz`. Have you tried explicitly specifying the model during the initialization of the Rasa engine?

The following minimal example works on my machine:

```python
from nl2ltl import translate
from nl2ltl.engines.rasa.core import RasaEngine
from nl2ltl.engines.utils import pretty

rasa = RasaEngine(model="/Users/francesco/Projects/nl2ltl/nl2ltl/engines/rasa/models/nlu-20230608-135823-concave-curve.tar.gz")

utterance = "Eventually send me a Slack after receiving a Gmail"
ltlf_formulas = translate(utterance, rasa)
pretty(ltlf_formulas)
```

**Comment by rtk29** [comment]
Hi @francescofuggitti,

Specifying the model during initialization worked. Thank you for that. However, I tried to see how the translation works with the same prompt used(i.e. the same above code), and it gives me an error like :
![image](https://github.com/IBM/nl2ltl/assets/127055206/f99e3dc5-fe29-46ec-8b35-5bb480736512)

After cloning the repo, I ran this piece of code in a jupyter notebook present in the same directory of the repo. Is there anything that I am missing?

Regards,
Raviteja.K
